recipes/adwaita-icon-theme/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "adwaita-icon-theme" %}
+{% set version = "3.36.0" %}
+{% set majmin_version = "3.36" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://download.gnome.org/sources/{{ name }}/{{ majmin_version }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 1a172112b6da482d3be3de6a0c1c1762886e61e12b4315ae1aae9b69da1ed518
+
+build:
+  number: 0
+  merge_build_host: False
+  ignore_run_exports:
+    - gtk3  # [win]
+    - librsvg  # [win]
+
+requirements:
+  build:
+    # gtk3 and librsvg here for rendering svg to png, build ok because platform independent
+    - gtk3  # [unix]
+    - librsvg  # [unix]
+    - make  # [unix]
+    - python
+  host:
+    # windows build files only know of one prefix, so these need to be in host even
+    # though we don't link against them and they are functionally build tools
```

Comment:
I'd suggest removing these and adding `merge_build_host: True # [win]`

Reply:
Good idea!